examples/ml_perf/main.py

```diff
@@ -0,0 +1,242 @@
+import argparse
+import importlib
+import os
+
+os.environ["KERAS_BACKEND"] = "jax"
+
+import keras
+
+import keras_rs
+
+from .dataloader import DataLoader
+from .model import DLRMDCNV2
+
+SEED = 1337
```

Comment:
It looks like you carefully passed the SEED everywhere, but it's hard to say if you missed one.

I would also do `keras.config.set_random_seed(SEED)` just to make sure things are consistent across hosts.